.github/workflows/build.yml

```
@@ -19,4 +19,84 @@
+                endpoint: "nanoframework"
+              }
+            ]
+          },
+          parameters: $parameters
+        }')
+
+        # Define the URL
+        url="https://dev.azure.com/${organization}/${project}/_apis/pipelines/${pipelineId}/runs?api-version=7.1"
+
+        # Trigger the pipeline
+        curl -s "${headers[@]}" -X POST -d "$bodyJson" "$url"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)